recipes/sithom/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "sithom" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e256e0afa95184a548e346405248ab5abc948b35f490f239591787389b07d09c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - cmocean
+    - jupyterthemes
+    - matplotlib-base
+    - python >=3.8
+    - seaborn
+    - typeguard
+    - uncertainties
+    - xarray
+
+test:
+  imports:
+    - sithom
```

```suggestion
test:
  imports:
    - sithom
  requires:
    - pip
  commands:
    - pip check
```
Typically we do something like this, but this would fail for your package because you have [pooch listed in your install_requires](https://github.com/sdat2/sithom/blob/0c44b637080cb9cedfcf2550190352ae553f12b5/setup.py#L20). You should remove pooch from those requirements if it is only required for tests.